From 566cebcf0ab26a3c36c3c34b76208f0dfda65bc8 Mon Sep 17 00:00:00 2001
From: Koichi Murase <myoga.murase@gmail.com>
Date: Thu, 30 Oct 2025 22:02:32 +0900
Subject: [PATCH] Lower the version requirement for GtkD temporarily as a
 workaround

We need to use GtkD-3.11.0, but even if we use the proper version of
the GtkD-3.11.0 library, the dependency check by the meson build fails
because GtkD-3.11.0 provides a wrong version information 3.10.0.  As a
workaround, we specify 3.10.0 as the version requirement, while
manually building and installing GtkD-3.11.0 from the upstream
repository in the CI.
---
 meson.build | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/meson.build b/meson.build
index f2d10be8..d790a897 100644
--- a/meson.build
+++ b/meson.build
@@ -96,8 +96,8 @@ tilix_sources = [
 sources_dir = include_directories('source/')
 
 # Dependencies
-gtkd_dep = dependency('gtkd-3', version: '>=3.11.0')
-vted_dep = dependency('vted-3', version: '>=3.11.0')
+gtkd_dep = dependency('gtkd-3', version: '>=3.10.0')
+vted_dep = dependency('vted-3', version: '>=3.10.0')
 xlib_dep = dependency('x11')
 libsecret_dep = dependency('libsecret-1', required: false)
 
